README.md

```diff
@@ -5,6 +5,10 @@ within a containerized environment, streamlining the process for continuous inte
 [OKD](https://okd.io/) payload.
 
 
+> [!NOTE]
+> You are advised to maximize available diskspace using an action like: https://github.com/marketplace/actions/maximize-build-disk-space
```

Comment:
Refer people to use a public action to accomplish the same if this is so essential.

Otherwise, people can run their own runners or pay for an Enterprise plan.